src/common/formatters.ts

```diff
@@ -134,6 +134,19 @@ export function getScheduleFrequencyString(schedule?: Admin.ISchedule) {
     if (schedule.rate) {
         return fixedRateToString(schedule.rate);
     }
+    if (schedule.cronSchedule && schedule.cronSchedule.schedule) {
+        return cronstrue.toString(`${schedule.cronSchedule.schedule}`);
```

Comment:
Is this string guaranteed to be in the correct format? Above, we're adding a leading 0 to ensure `cronstrue` can parse it correctly.

Reply:
Also, we don't need the formatting (`${}`) if we're just passing the value along?

Reply:
At least in flytekit we verify it is a valid Unix flavoured cron expression. I noticed the padding 0 above to deal with AWS cron but that is not assured either and mostly replies on contract. Maybe a better way is to handle the exception and display something indicating invalid cron? Now I realize it could be an alias too like `daily`.

Reply:
Valid point about it just being convention/contract. I think my question was whether or not the expected value _should_ be a normal cron string based on convention. If we're ensuring that in flytekit, that's equivalent to the existing case.

Reply:
I will add additional logic to handle aliases.